### DISCUSSION TITLE
Repeater shift function Side Key

### DISCUSSION THREAD

**Discussion by f5gkw** [egzumer/uv-k5-firmware-custom#131] [Ideas]
First off all, lot of thanks for the job !
Superb AM demodulation improvment !

Does it possible to add an option, for the side keys or long press M key, to permit to switch quickly in simplex or duplex mode ? 

Thanks, Jean-Phi, very new owner of the UV-K5

**Comment by ErikS-web** [comment]
Do you mean. The functionality of [F+ `R`](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Button-functions#8-r)

**Comment by f5gkw** [comment]
Hi.
No, I don't.
I know what is the reverse function ( F+8)  : to listen to the input
frequency of a repeater. (and to transmit on the repeater output frequency
" around me" , ).

What I would like : If I key 145.650 which is a repeater ouptut frequency,
I need to go in the nenu to declare a shift operation, and the direction.
Oftenly on vhf, the direction is - ( minus) and offset 0.6 Mhz. So, now I
am able to use duplex (instead of simplex operation).

Now, if I key 145.500 which is a simplex frequency, I need to go back to
the menu to remove the repeater function..
I would like a more faster way to enable/disable repeater operation.

Regards,
JPh aka F5GKW

**Reply by egzumer** [reply]
I don't think this is that useful. There are many other options that you need to setup too. Not every option needs it's own button function. Apart from saving your settings to memory channel you can also use upper and lower VFO, they have separate settings memory, you can use upper for simplex and lower for duplex.

**Comment by ErikS-web** [comment]
A **Memory channel** is ideal for your question. It remembers the shift setting and more. And then you can simply go back to VFO. There are 200 memory channels
I have included all national repeaters in my radio. You can make such a list with [Chirp.](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Radio-operation#memory-channels-scanning)

**Tip**: With [F+9 `Call`](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Button-functions#9-call) you can quickly go to a _specific memory channel_ you have set, such as _your local repeater_.

**Comment by f5gkw** [comment]
hi,
You are right.
I am probably"old school" and I don't use a lot the memories.

the mess arrives when you visit an aerea with multiple UHF repeaters (
shift direction not always +, offset can be 1.6 , 7.6 or 9.4 .. ) and of
course another mess on V/UHF repeaters. and ctcs or not ... you can't place
all that in memories.

73 !
JPh F5GKW


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Sans
virus.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by f5gkw** [comment]
Hi,
Thanks.
As I have answered, I am probably a "too old school" ham, and I don't
enough think about memories usages..
You can close the subject.
J-Phi

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Sans
virus.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
